vector-im/element-android
# #5581: live location sharing feature flag

## Type of change


## Content


Adding a way to enable/disable the live location feature from private settings.

## Motivation and context


Closes #5581 

## Screenshots / GIFs



## Tests



- Go to private debug options
- Click on features
- Enable the live location feature
- Go to a room
- Start sharing your location
- Check the live location sharing is visible when the pin is on the current user location
- Go to private debug options
- Click on features
- Disable the live location feature
- Go to a room
- Start sharing your location
- Check the live location sharing is not visible when the pin is on the current user location

## Tested devices

- OS version(s): Android 12

## Checklist
